src/test/java/org/openelisglobal/integrationTests/referenceTables/ReferenceTablesServiceTest.java

```diff
@@ -0,0 +1,96 @@
+package org.openelisglobal.integrationTests.referenceTables;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.openelisglobal.BaseTestConfig;
+import org.openelisglobal.config.TestConfig;
+import org.openelisglobal.referencetables.service.ReferenceTablesService;
+import org.openelisglobal.referencetables.valueholder.ReferenceTables;
+import org.openelisglobal.testUtils.ObjectsInitiator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(classes = { BaseTestConfig.class, TestConfig.class })
+@TestPropertySource("classpath:common.properties")
+@ActiveProfiles("test")
+public class ReferenceTablesServiceTest {
+
+    @Autowired
+    private ReferenceTablesService referenceTablesService;
+
+    private String id;
+
+    private ReferenceTables expected;
+    private ReferenceTables result;
+
+
+    @Test
+    public void insert_shouldInsertReferenceTable(){
+        expected = ObjectsInitiator.createReferenceTable("X-Table","N","Y");
+        id = referenceTablesService.insert(expected);
+
+        result = referenceTablesService.get(id);
+        assertEquals(expected.getIsHl7Encoded(),result.getIsHl7Encoded());
```

Comment:
Could you create a method that does assertEquals for every field except id that we can re-use between tests? Ideally we would like to know that all fields are entered successfully on insert, not just ishl7Encoded